src/core/session.ts

```diff
@@ -34,6 +34,7 @@ export class Session implements FormSubmitObserverDelegate, HistoryDelegate, Lin
 
   readonly frameRedirector = new FrameRedirector(document.documentElement)
 
+  drive = true
   enabled = true
```

Comment:
@gregschmit I'm sorry to dig up an old pull request, but could you help me understand why a `drive` property was introduced alongside [enabled](https://github.com/gregschmit/turbo/blame/67f068edefaa865e0eda78131231521d3513d609/src/core/session.ts#L38)? Could `enabled` be repurposed to serve the same purpose?

Reply:
I might need to look at the code, but my thinking is that `enabled` on `Turbo.session` enabled/disabled everything (or the session?), whereas this feature was designed to specifically disable `Drive` behavior until enabled in the document using `data-turbo="true"`, allowing opt-in behavior. So with `Turbo.session.drive = false`, everything is still enabled (especially frames/streams), just that drive must now be opted-into rather than opted-out-of (making the behavior similar to frames/streams).

Does that make sense? I can still take a look at the code and see if i'm wrong about my assumptions of what `enabled` does.

BTW, this is the confusion I was hoping to mitigate by naming the property `disableDriveByDefault` but it was considered too verbose.